mvc/overview.rst

```diff
@@ -1,8 +1,65 @@
-.. include:: /../common/stub-topic.txt
+Overview of ASP.NET MVC
+=======================
+By `Tom Archer`
 
-|stub-icon| Overview of ASP.NET MVC
-===================================
+Based on the *MVC architectural pattern*, the ASP.NET MVC framework is a lightweight, cross-platform, highly testable application-development framework that separates an application into three main components: model, view, and controller:
 
-.. include:: /../common/stub-notice.txt
+- :doc:`Models </models/index>` are the parts of the application that implement the logic for the application's data domain. Often, model objects retrieve and store model state in a database. For example, a Product object might retrieve information from a database, operate on it, and then write updated information back to a Products table in a SQL Server database.
```

Comment:
per danroth, Use the https://msdn.microsoft.com/en-us/library/dd381412(v=vs.108).aspx here. Include the image.